src/bubble-chart.js

```diff
@@ -123,9 +128,9 @@ dc.bubbleChart = function (parent, chartGroup) {
         // override default x axis brush from parent chart
     };
 
-    _chart.redrawBrush = function () {
+    _chart.redrawBrush = function (g, selection, doTransition) {
```

Comment:
Whoops - looks like you added some arguments here but never pass them anywhere. I just ran into this because I wanted to add a `dontSetZoom` argument. Do you remember what you were trying to do here?

Reply:
Specifically, it looks like you changed the signature to take the current filter/selection, but this also takes `g` and `doTransition` - so I guess it's a simple fix since those are unused.